src/query/config/src/outer_v0.rs

```diff
@@ -298,6 +306,68 @@ impl TryInto<InnerStorageConfig> for StorageConfig {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Serialize, Deserialize)]
+pub struct CacheConfig {
+    #[serde(rename = "type")]
+    pub cache_type: String,
+
+    #[serde(rename = "num_cpus")]
```

Comment:
This config used for?

Reply:
not used so far. Maybe we can remove it.